api/src/config.py

```diff
@@ -11,7 +11,7 @@ class DevConfig(Config):
 
 class TestConfig(Config):
     SQLALCHEMY_DATABASE_URI = os.environ.get('TEST_DATABASE_URL') or \
-        'sqlite://'
+        'sqlite:///test.db'
 
```

Comment:
This may be a non-issue, since we are always planning on having a database container running, but I am not sure what will happen if ```TEST_DATABASE_URL``` is not defined and ```os.environ.get``` returns ```None```, causing it to look for ```test.db```, which currently does not exist. If it auto-creates ```test.db``` we are fine. The ```sqlite://``` option uses an in-memory database, which doesn't require a file to be defined beforehand.

Reply:
Addressing this is less important than other comments.